### PR TITLE
Add auto complete examples

### PR DESCRIPTION
**Pull request by JeffGreiner-eaton**
Fixes BLUI-5989.

#### Changes proposed in this Pull Request:

- Add auto complete code snippets
- Hide auto complete playground
-

#### Screenshots / Screen Recording (if applicable)

image

#### To Test:

-

#### Any specific feedback you are looking for?

-

### CODE REVIEW

File: docs/src/componentDocs/AutoComplete/examples/AutoComplete.tsx [hunk 1]
```diff
@@ -0,0 +1,22 @@
+import React from 'react';
```
[the remark this answers is not in the document]
Comment (JeffGreiner-eaton) [reply docs/src/componentDocs/AutoComplete/examples/AutoComplete.tsx:1]: yes in the component. here it is on iOS


https://github.com/user-attachments/assets/ea772e2f-d934-4327-8691-3097b99e2e8b

File: docs/src/componentDocs/AutoComplete/examples/AutoComplete.tsx [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+import { AutoCompleteBasicExample } from './AutoCompleteBasicExample';
+
+const codeSnippet = `<AutoComplete
+    helperText="Helper text"
+    label="Popular Cities"
+    value={['New York']}
+    options={['Dubai','London','New York','Paris']}
+/>`;
+
+export const AutoComplete = (): JSX.Element => (
```
[the remark this answers is not in the document]
Comment (JeffGreiner-eaton) [reply docs/src/componentDocs/AutoComplete/examples/AutoComplete.tsx:13]: Do you see something wrong with basic usage?
- <AutoComplete helperText="helper text" label="label" value={['Tag1']} options={['Tag1', 'Tag2', 'Tag3', 'Tag4']} />

[the remark this answers is not in the document]
Comment (JeffGreiner-eaton) [reply docs/src/componentDocs/AutoComplete/examples/AutoComplete.tsx:13]: But I guess we can replace example code with...something like

image

wrong image but looks like this
image

File: docs/src/componentDocs/AutoComplete/examples/AutoComplete.tsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (JeffGreiner-eaton) [reply docs/src/componentDocs/AutoComplete/examples/AutoComplete.tsx:1]: issue opened https://github.com/etn-ccis/blui-react-native-component-library/issues/615

File: docs/src/componentDocs/AutoComplete/examples/AutoComplete.tsx (hunk 2 above)
[the remark this answers is not in the document]
Comment (joebochill) [reply docs/src/componentDocs/AutoComplete/examples/AutoComplete.tsx:13]: The component doesn't work in the web...so we don't want to show any interactive examples.

File: docs/src/componentDocs/AutoComplete/examples/AutoComplete.tsx [hunk 3]
```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import Box from '@mui/material/Box';
+import { CodeBlock, CodeBlockActionButtonRow } from '../../../shared';
+import { AutoCompleteBasicExample } from './AutoCompleteBasicExample';
+
+const codeSnippet = `<AutoComplete
+    helperText="Helper text"
```
[the remark this answers is not in the document]
Comment (joebochill) [reply docs/src/componentDocs/AutoComplete/examples/AutoComplete.tsx:7]: Double check this in all of the snippets.

**Review by joebochill — APPROVED** [review]

**Pull request merged**